src/components/tabController/PageCarousel.tsx

```diff
@@ -23,16 +23,16 @@ class PageCarousel extends PureComponent {
   componentDidMount() {
     if (Constants.isAndroid) {
       setTimeout(() => {
-        this.scrollToPage(this.context.selectedIndex, false);
+        this.scrollToPage(this.context.selectedIndex);
       }, 0);
     }
   }
 
-  onTabChange = ([index]) => {
-    this.scrollToPage(index, true);
+  onTabChange = ([index]: readonly number[]) => {
+    this.scrollToPage(index);
   };
 
-  scrollToPage = (pageIndex, animated) => {
+  scrollToPage = (pageIndex: number) => {
```

Comment:
It seems like you changes the functionality here. 
Why did you remove the `animated` parameter?

Reply:
It was not used, `false` was always sent (and I had an error about it).

Reply:
ok.